packages/mds-compliance/api.ts

```diff
@@ -195,11 +206,19 @@ function api(app: express.Express): express.Express {
         return [...acc, getPolygon(geographies, geography.geography_id)]
       }, [])
 
+      // Reading all devices since this is across all providers.
+      const deviceRecords = await db.readDeviceIds()
+      const deviceIDs = deviceRecords.map(recorded_device => recorded_device.device_id)
```

Comment:
I don't think we should do these reads unless we're operating in a 'historical' context